sys/include/net/netstats.h

```diff
@@ -18,13 +18,22 @@
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
  */
 
+#include <stdint.h>
+
 #ifndef NETSTATS_H
 #define NETSTATS_H
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+typedef enum {
+    NETSTATS_LAYER2     = 0x01,
+    NETSTATS_IPV6       = 0x02,
+    NETSTATS_ALL        = 0xFF
```

Comment:
Okay, maybe this was more of a signed/unsigned issue.... Can't remember, just that we had (casting related) issues with flags defined in enums in the past, that yielded compiler warnings.